app/views/peep.erb

```diff
@@ -0,0 +1,9 @@
+<li class='peep'>
+  <div class='content'>
+    <%= peep.content %>
+  </div>
+  <span class='source'>
+    peeped at <%= peep.created_at.to_datetime.strftime("%l:%M%p %A %e %B") %> by <%= User.get(peep.user_id).username %>
```

Comment:
You should never be calling your models directly from the view! Also, multiply chained methods like this are a definite code smell - check out the Law of Demeter